recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
+        - paste
+        - pillow
+        - python-dateutil
+        - pytz
+        - requests
+        - scipy
+        - six
+        - tk 8.5.*
+    run:
+        - python
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
+        - paste
+        - pillow
+        - python-dateutil
+        - pytz
+        - requests
+        - scipy
+        - six
+        - tk 8.5.*
+        # FIXME: Workaround until https://github.com/conda-forge/matplotlib-feedstock/issues/36 is fixed.
```

Comment:
> I want to commit your yaml changes, like this:

That is fine, but the commit messages on staged-recipes are not "important" b/c once merged this recipe will be moved to its own repository (feedstock). Do you want to add that as comment somewhere? Or are you OK merging this as is and adding that comment as an issue so you can keep track of possible problems in the future (and change the specified versions)?